spec/outcome_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe Outcome do
+
+  it "should check for all types of horizontal wins" do
```

Comment:
Remember how I've previously said that using "check" as a verb is ambiguous? Same goes for descriptions. This should read `it "should return true for all types of horizontal wins"`, etc.

Reply:
Yes! Thanks for pointing that out :)